database-commons/src/main/java/io/cdap/plugin/db/connector/AbstractDBSpecificConnectorConfig.java

```diff
@@ -136,4 +136,9 @@ public int getPort() {
   public String getTransactionIsolationLevel() {
     return null;
   }
+
+  public boolean connectionParamsContainsMacro() {
```

Comment:
can we just call this canConnect()?

Reply:
Similar comment here as well, its being used by the base class AbstractDBSink, renaming here will lead to method not being recognised there, 
https://github.com/data-integrations/database-plugins/blob/873f86ad337403dadd824607d692ba3524d2dbce/database-commons/src/main/java/io/cdap/plugin/db/batch/sink/AbstractDBSink.java#L104